packages/i18n-calypso/test/index.js

```diff
@@ -0,0 +1,376 @@
+/**
+ * External dependencies
+ */
+var assert = require( 'assert' ),
+	React = require( 'react' ),
+	ReactDomServer = require( 'react-dom/server');
+
+/**
+ * Internal dependencies
+ */
+var data = require( './data' ),
+	i18n = require( '..' ),
+	moment = i18n.moment,
+	numberFormat = i18n.numberFormat,
+	translate = i18n.translate;
+
+/**
+ * Pass in a react-generated html string to remove react-specific attributes
+ * to make it easier to compare to expected html structure
+ * @param  {string} string React-generated html string
+ * @return {string}        html with react attributes removed
+ */
+function stripReactAttributes( string ) {
+	return string.replace( /\sdata\-(reactid|react\-checksum)\=\"[^\"]+\"/g, '' );
+}
+
+describe( 'I18n', function() {
+	beforeEach( function() {
+		i18n.setLocale( data.locale );
+	} );
+
+	afterEach( function() {
+		i18n.configure(); // ensure everything is reset
+	} );
+
+	describe( 'setLocale()', function() {
+		describe( 'adding a new locale source from the same language', function () {
+			beforeEach( function() {
+				i18n.setLocale( {
+					'': data.locale[''],
+					'test1': [
+						null,
+						'translation1-1'
+					],
+					'test2': [
+						null,
+						'translation2'
+					],
+					'new translation': [
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'New Translation Set' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).